apps/full-site-editing/full-site-editing-plugin/block-inserter-modifications/contextual-tips/style.scss

```diff
@@ -1,3 +1,3 @@
-.block-editor-inserter__panel-content .contextual-tip {
-	margin-top: 20px;
+.contextual-tip {
+	margin: 20px;
```

Comment:
The default tips have [a margin of 16px (`$grid-unit-20`)]( https://github.com/WordPress/gutenberg/blob/9f45d0baf6bead528a50829030c795cc5e2ec585/packages/block-editor/src/components/inserter/style.scss#L246), could we align those two?